cnf-certification-test/accesscontrol/suite.go

```diff
@@ -649,3 +655,110 @@ func Test1337UIDs(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", badPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+type ContainerSCC struct {
+	HostDirVolumePlugin bool
+	HostIPC             bool
+	HostNetwork         bool
+	HostPID             bool
+	HostPorts           bool
+	PrivilegeEscalation bool
+	PrivilegedContainer bool
+	RunAsUser           bool
+	Capabilities        string
```

Comment:
the following seem missing:
- readOnlyRootFilesystem
-runAsGroup
-runAsNonRoot
-seLinuxOptions